ui/component/channelEdit/view.jsx

```diff
@@ -284,39 +297,52 @@ function ChannelForm(props: Props) {
+    setTabIndex(index);
+  }
+
   // TODO clear and bail after submit
   return (
     <>
       <div className={classnames({ 'card--disabled': disabled })}>
-        <header className="channel-cover">
-          <div className="channel__quick-actions">
-            <Button
-              button="alt"
-              title={__('Cover')}
-              onClick={() =>
-                openModal(MODALS.IMAGE_UPLOAD, {
-                  onUpdate: (coverUrl, isUpload) => handleCoverChange(coverUrl, isUpload),
-                  title: __('Edit Cover Image'),
-                  helpText: __('(6.25:1 ratio, %max_size%MB max)', {
-                    max_size: THUMBNAIL_CDN_SIZE_LIMIT_BYTES / (1024 * 1024),
-                  }),
-                  assetName: __('Cover Image'),
-                  currentValue: params.coverUrl,
-                })
-              }
-              icon={ICONS.CAMERA}
-              iconSize={18}
-            />
-          </div>
-          {params.coverUrl &&
-            (coverError && isUpload.cover ? (
+        <header className="channel-cover" style={{ backgroundImage: 'url(' + coverSrc + ')' }}>
+          <div className="channel-header-content">
+            <div className="channel__quick-actions">
+              <Button
+                button="alt"
+                title={__('Cover')}
+                onClick={() =>
+                  openModal(MODALS.IMAGE_UPLOAD, {
+                    onUpdate: (coverUrl, isUpload) => handleCoverChange(coverUrl, isUpload),
+                    title: __('Edit Cover Image'),
+                    helpText: __('(6.25:1 ratio)', {
+                      max_size: THUMBNAIL_CDN_SIZE_LIMIT_BYTES / (1024 * 1024),
+                    }),
+                    assetName: __('Cover Image'),
+                    currentValue: params.coverUrl,
+                    otherValue: params.thumbnailUrl,
+                  })
+                }
+                icon={ICONS.CAMERA}
+                iconSize={18}
+              />
+            </div>
+            {params.coverUrl && coverError && isUpload.cover && (
               <div className="channel-cover__custom--waiting">
                 <p>{__('Uploaded image will be visible in a few minutes after you submit this form.')}</p>
               </div>
-            ) : (
-              <img className="channel-cover__custom" src={coverSrc} onError={() => setCoverError(true)} />
-            ))}
-          <div className="channel__primary-info">
+            )}
+            <div className="channel__primary-info">
+              <h1 className="channel__title">
+                {params.title || (channelName && '@' + channelName) || (params.name && '@' + params.name)}
+              </h1>
+            </div>
+          </div>
+        </header>
+
+        <Tabs index={tabIndex}>
+          <div className={classnames('tab__wrapper', { 'tab__wrapper-fixed': scrollPast })}>
             <div className="channel__edit-thumb">
```

Comment:
<details><summary>Thumbnail edit button affected by screen zoom level? (tested on Chrome) on default value (100%) it's not clickable</summary>

![chrome_AKSn12Y6Rz](https://user-images.githubusercontent.com/76502841/199079313-91e71a6e-32b3-4c44-a37d-4802773e3779.gif)</details>